src/formatter.coffee

```diff
@@ -41,6 +41,7 @@ class Formatter extends SimpleModule
       h2: ['margin-left', 'text-align']
       h3: ['margin-left', 'text-align']
       h4: ['margin-left', 'text-align']
+      input: ['width']
```

Comment:
I think this is not necessary now since we are resolving the width of the input via style attr.

Reply:
Same as above

Reply:
you are right, I didn't notice that you are specify style attributes.